website/docs/Installation/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Installation Guide for Podman Desktop
+---
+
+Welcome to the Installation Guide for Podman Desktop!
+
+Podman Desktop is available across all platforms and this documentations intends to help you install the latest version of Podman Desktop on your Operating System. The links to installation guide for different platforms are listed below.
```

Comment:
I think 'all platforms' is misleading as it's only Mac arm64/x64, win x64 and Linux x64
But all platforms include several other operating systems / arches

Reply:
should I replace it with "across Windows, MacOS and Linux"?

Reply:
maybe

Reply:
done